controllers/openshift_controller.go

```diff
@@ -314,6 +395,9 @@ func (r *KataConfigOpenShiftReconciler) newMCPforCR() (*mcfgv1.MachineConfigPool
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name: "kata-oc",
+			Labels: map[string]string{
+				"crio-conf": "",
```

Comment:
@pmores two requests:
1. Can you please add a comment mentioning the need for MCP label as ContainerRuntimeConfig expects an MCP label selector
2. How about using the MCP name itself as label - `kata-oc` instead of `crio-conf`. Better make the MCP name as constant and replace it for the entire code.

Reply:
@bpradipt 
ad 1) sure!
ad 2) can do. My original thought was that labels are cheap so why not call the label to express what it's for (it's another thing though whether 'crio-conf' is the best way to do that ;-)).  That way, even if, say, we end up with multiple MCPs in the future we'd just label the right one with no other changes required.  But if we don't think we'll be handling multiple MCPs then using the MCP name makes sense, and there's certain consistency in doing so.

Reply:
@pmores I'm thinking from the admin pov and feel keeping the labels consistent with existing patterns will be useful.  How about we just use this `"pools.operator.machineconfiguration.openshift.io/kata-oc: ""`.
Similar to existing MCP labels, but it's not mandatory. I'll leave it to your choice :-)

Reply:
@bpradipt Alright then, I'll fix it!